tailcfg/tailcfg_test.go

```diff
@@ -571,6 +571,17 @@ func TestNetInfoFields(t *testing.T) {
 		"PreferredDERP",
 		"LinkType",
 		"DERPLatency",
+		"DERPLatencyV4",
+		"DERPLatencyV6",
+		"UDP",
+		"IPv6",
+		"IPv4",
+		"IPv6CanSend",
+		"IPv4CanSend",
+		"ICMPv4",
+		"GlobalV4",
+		"GlobalV6",
+		"CaptivePortal",
```

Comment:
why is this change needed?

Reply:
If you run `go generate ./...`, you get this test fails: https://github.com/coder/tailscale/blob/d8574de4b4ca468d8c631be581d095650b31426f/tailcfg/tailcfg_test.go#L559-L559

The change is only in a test, that validates the `Clone()` method is missing some fields. These fields were omitted in the test that we added here: https://github.com/coder/tailscale/pull/56

I was just getting a test to pass that was failing locally.